.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please, do not commit IDE-related files. So please remove `.idea` directory
I suggest to use `git commit -u` instead of `git commit -A` to avoid checking in unintended files.